torch/nested/_internal/ops.py

```diff
@@ -2212,6 +2212,41 @@ def new_empty_default(func, *args, **kwargs):
     raise RuntimeError("new_empty() not supported for NJT with shape != ()")
 
 
+@register_jagged_func(
+    [
+        torch.ops.aten.elu_backward.default,
+        torch.ops.aten.hardshrink_backward.default,
+        torch.ops.aten.hardsigmoid_backward.default,
+        torch.ops.aten.hardtanh_backward.default,
+        torch.ops.aten.softplus_backward.default,
+        torch.ops.aten.softshrink_backward.default,
+    ],
+    "self: jt_all, ...",
+)
+def activation_backward(func, *args, **kwargs):
+    # first NJT arg is expected to be grad_output
+    grad_output = next(arg for arg in args if isinstance(arg, NestedTensor))
```

Comment:
Add an. assert that arg is not empty?

Reply:
`check_schema()` handles this part :)

Reply:
But couldn't none of the args be NestedTensors?

Reply:
We wouldn't have reached this torch dispatch if none are NestedTensors

Reply:
`check_schema()` will validate that `self` is an NJT so we're guaranteed at least one, plus we're inside of `NestedTensor.__torch_dispatch__()` so we had to get there somehow from a `NestedTensor` instance